.happy/terraform/modules/wmg-batch/main.tf

```diff
@@ -0,0 +1,53 @@
+# This is a batch job
```

Comment:
An alternative to creating this batch job is to parameterized the one that we already have. In general, I think this solution is better, since they're two distinct jobs (this one will require less variables, for instance). Code duplication is quite unavoidable with Terraform anyway.

We should probably modify the other module to be called `upload-batch` though.

Reply:
I think keeping them as 2 separate jobs makes sense to me. Agree re renaming the other job